samples/benchmark/gamelife/kotlin/src/main/kotlin/godot/samples/benchmark/gamelife/game.kt

```diff
@@ -0,0 +1,176 @@
+package godot.samples.benchmark.gamelife
+
+import godot.*
+import godot.core.*
+import org.godotengine.kotlin.annotation.RegisterClass
+import org.godotengine.kotlin.annotation.RegisterFunction
+import org.godotengine.kotlin.annotation.RegisterProperty
+import kotlin.math.max
+import kotlin.math.sqrt
+
+@RegisterClass("scripts/gamelife")
+class Game: MultiMeshInstance(){
+
+    private lateinit var map : Map
+    private var started: Long = 0
+
+    private var counter : Int = 0
+    private var frame : Int = 0
+    private val rng = RandomNumberGenerator()
```

Comment:
Personally I'm no fan of shortened variable names as it's bad for readability in code later on. But as this benchmark is one single script and is not complicated, you can leave it like it is if you want to.
Otherwise i would prefer `randomNumberGenerator` as variable name